controllers/common.go

```diff
@@ -0,0 +1,10 @@
+// Copyright Contributors to the Open Cluster Management project
+package controllers
+
+func getLabels(val string) map[string]string {
```

Comment:
Can you add a comment describing this function?  Maybe a better name is `generateLabels()`?